dashboard/config/libraries/NativeSpriteLab.interpreted.js

```diff
@@ -136,36 +136,36 @@ function setupSim(
   for (counter_i = 0; counter_i < s3number; counter_i++) {
     makeNewSpriteAnon(s3costume, randomLocation());
   }
-  setProp(s3costume, 'scale', 50);
+  setProp({costume: s3costume}, 'scale', 50);
 
   for (counter_i = 0; counter_i < s2number; counter_i++) {
     makeNewSpriteAnon(s2costume, randomLocation());
   }
-  addBehaviorSimple(s2costume, new Behavior(movementBehavior(s2speed)));
+  addBehaviorSimple({costume: s2costume}, new Behavior(movementBehavior(s2speed)));
 
   for (counter_i = 0; counter_i < s1number; counter_i++) {
     makeNewSpriteAnon(s1costume, randomLocation());
   }
-  addBehaviorSimple(s1costume, new Behavior(movementBehavior(s1speed)));
+  addBehaviorSimple({costume: s1costume}, new Behavior(movementBehavior(s2speed)));
```

Comment:
Just wanted to make sure that the change from s1speed -> s2speed was intentional.

Reply:
It was not! Thanks for catching